frontend_app_gallery/src/container/Gallery.jsx

```diff
@@ -181,11 +183,9 @@ class Gallery extends React.Component {
 
     switch (fetchContentList.apiResponse.status) {
       case 200:
-        let images = []
-        fetchContentList.body.forEach(content => {
-          if (content.content_type === 'file') {
-            images.push({ src: '', contentId: content.content_id })
-          }
+
+        const images = fetchContentList.body.map(content => {
```

Comment:
this doesn't work.
It will fill your returned array with undefined values for non file contents. A .map() must always return a value or undefined will be put in the returned array

you must filter your array before doing the .map()

``` javascript
const images = fetchContentList.body.filter(c => c.content_type === 'file').map(c => ({ src: '', contentId: c.content_id }))
```